config/general.php

```diff
@@ -21,6 +21,7 @@
     ->preloadSingles()
     // Prevent user enumeration attacks
     ->preventUserEnumeration()
+    ->limitAutoSlugsToAscii()
```

Comment:
Other preferences have a comment briefly describing it.


```suggestion
    // Convert non-ASCII to ASCII characters in auto-generated slugs (i.e. ñ → n).
    ->limitAutoSlugsToAscii()
```